### PR TITLE
test: Enable GRPC for ITKmsTest

### PR DESCRIPTION
**Pull request by sydney-munro**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-storage/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java [hunk 1]
```diff
@@ -274,45 +298,52 @@ private static String ensureKmsKeyExistsForTests(
...
   public void testClearBucketDefaultKmsKeyName() throws ExecutionException, InterruptedException {
-    String bucketName = dataGeneration.getBucketName();
+    String bucketName = bucketFixture.newBucketName();
     Bucket remoteBucket =
-        storage.create(
-            BucketInfo.newBuilder(bucketName)
-                .setDefaultKmsKeyName(kmsKeyOneResourcePath)
-                .setLocation(KMS_KEY_RING_LOCATION)
-                .build());
+        storageFixtureHttp
```
Comment (BenWhitehead) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java:303]: Can we add todo's with the bug number next to these explicit http usages to cleanup and use `storage` once it's available?

**Review by BenWhitehead — APPROVED** [review]
One small comment request

File: google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java (hunk 1 above)
Comment (sydney-munro) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java:308]: yeah I can do that, I was kinda just planning to search for storageFixtureHttp and remove the uses one by one but have no real preference.

Comment (BenWhitehead) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java:308]: That's probably fine as long as we're tracking it somewhere

**Review by BenWhitehead — APPROVED** [review]

**Pull request merged**